src/main/scala/com/github/unchama/seichiassist/minestack/MineStackObject.scala

```diff
@@ -0,0 +1,48 @@
+package com.github.unchama.seichiassist.minestack
+
+import org.bukkit.Material
+import org.bukkit.inventory.ItemStack
+
+case class MineStackObject(
+  mineStackObjectName: String,
+  uiName: Option[String],
+  itemStack: ItemStack,
+  hasNameLore: Boolean, // 多分記名があるかどうか
+  category: MineStackObjectCategory
+) {
+
+  def getMaterial: Material = itemStack.getType
+
+  def getDurability: Short = itemStack.getDurability
```

Comment:
```suggestion
  def material: Material = itemStack.getType

  def durability: Short = itemStack.getDurability
```

`get` は不要そう(uniform access principleによる)